spk/salt-master/Makefile

```diff
@@ -1,38 +1,53 @@
 SPK_NAME = salt-master
-SPK_VERS = 2015.5.11
-SPK_REV = 9
+SPK_VERS = 3004
+SPK_REV  = 10
 SPK_ICON = src/salt-master.png
 
-DEPENDS =
-WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>2.7.8-10"
+SPK_DEPENDS = "python38"
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
-RELOAD_UI = yes
-DISPLAY_NAME = Salt Master
-CHANGELOG = "1. Update to 2015.5.11<br/>2. Fix salt-api<br/>3. Update dependencies"
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+BUILD_DEPENDS += cross/wheel
```

Comment:
again, python310.  And with the various changes there is no need to include setuptools, pip or wheel.